Makefile

```diff
@@ -408,38 +408,30 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=0.11.0
+protoVer=0.11.2
```

Comment:
As I have modified the dockerfile, bumping to 0.11.2.
The person merging must NOT forget to trigger the image build